### ISSUE TITLE
missing several networks from ustvgo

### ISSUE THREAD

**Issue by blealolealoleal** [Spaxly/tv-cli#1]
there's 102 networks on ustvgo. this scrapes from there but only has 56 for some reason.

**Comment by Spaxly** [comment]
I am aware of this and taking a look at this issue.

**Comment by pitsi** [comment]
It probably does not help with the question, but 25 of the 102 channels need vpn to work (and not just any vpn, nordvpn).
Nba tv for instance is one of them, and I am glad the script does not include it. 
This means that there are 77 more channels which work without vpn, so yes, the script is missing 21.

**Comment by Spaxly** [comment]
Yup, some channels do require a VPN, i'm guessing due to legal reasons. I may have a solution to this though and currently working on it.

**Comment by pitsi** [comment]
Is it just me or some more (or maybe all) channels now require the vpn? I tried playing a lot of them, both from the script and from the page as well, but all of them returned the "no hls link found" error (for the script) or the "Please change your ip address to the US with your VPN" prompt (for the page).
Obviously, I do not have a US ip...

**Comment by pitsi** [comment]
So I checked every channel. Out of the 102, only 3 (cnn, fox news and msnbc) work without the vpn!

**Comment by Spaxly** [comment]
Unfortunately there is nothing on my side that I can do to fix this as it involves geo-blocking.

**Comment by pitsi** [comment]
I know and I respect it. However, it would be nice if someone with a US ip can verify that the site can show all those channels without the vpn. If not, I do not think it is worth maintaining the script just for 3 channels.

**Comment by Spaxly** [comment]
I am in the US and all channels work here without a VPN.

**Comment by pitsi** [comment]
Great! This means that I either find a way to "plant" a US proxy on mpv or I delete the script, bucause those 3 channels that can work witnout a US ip are not what I would watch.

Thank you for the feedback :)

**Comment by Spaxly** [comment]
> Great! This means that I either ~find a way to "plant" a US proxy on mpv~ or I delete the script, bucause those 3 channels that can work witnout a US ip are not what I would watch.
> 
> Thank you for the feedback :)
> 
> ---edit Ignore that thing I striked out. It is not mpv that gets the urls, curl is.

I will take a look at adding a site that is more efficient and does not have geo-restrictions

**Comment by pitsi** [comment]
There is no need to. No other site matches the reliability of ustvgo when it comes to stable streaming.
If all its content is available to US users (or users with a US ip) without a vpn since ever, it is just the site that imposes bigger restrictions to non-US users time after time... 
And despite using it for almost a year, I learned about it just yesterday, from your comment.

**Comment by Spaxly** [comment]
There are many other streaming sites that provide more channels and no geo-restrictions and have more stability then ustvgo. I will be adding this to the readme as well.
